cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1374,6 +1380,12 @@ public void run() {
         if (_executionTimerInvolveBrokerRemovalOrDemotion != null) {
           _executionTimerInvolveBrokerRemovalOrDemotion.update(duration, TimeUnit.MILLISECONDS);
         }
+
+        OPERATION_LOG.info("Execution Finished: task Id: {}; removed brokers: {}; demoted brokers: {}; total time used: {}ms.",
+                           _userTaskInfo != null ? _userTaskInfo.userTaskId() : "N/A (triggered by self-healing)",
```

Comment:
I thought there was always a task ID, even for self-healing? I'm pretty sure the emails include one, and the existing logs certainly do.

Reply:
Interesting. Let me double check.

Reply:
You are completely right. Looks like uuid always exists while userTaskInfo does not. Thanks for the catch!